src/App.jsx

```diff
@@ -1,34 +1,71 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import { GlobalStyles } from "./styles/GlobalStyles";
+import { getDate } from "./utils/helpers";
+import { useState } from "react";
+import Wrapper from "./components/Wrapper";
+import Heading from "./components/Heading";
+import DateParagraph from "./components/DateParagraph";
+import Input from "./components/Input";
+import List from "./components/List";
 
 function App() {
-  const [count, setCount] = useState(0)
-
-  return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
-  )
+	const [tasks, setTasks] = useState([
+		{ id: 1, title: "Fake task from state", done: false },
+		{ id: 2, title: "Fake task from state", done: true },
+	]);
+
+	const changeTaskStatus = (index) => {
+		tasks[index].done = !tasks[index].done;
```

Comment:
Ja bym tutaj jednak napisał to raczej na zasadzie odniesienia się do prevState i wykonania akcji na tym, poprzez filtrowanie, bezpieczniej.